src/tools/debugger/FrontendConnector.java

```diff
@@ -207,18 +209,53 @@ public void awaitClient() {
   private static Map<SFarReference, String> createActorMap(
```

Comment:
What's the difference between `createActorMap` and `createObjMap`?
Do we really need both?